ironfish/src/serde/Serde.ts

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Moved this from the serde/index.ts because VSCode won't pick up a better import suggestion otherwise, plus it just makes sense given the issue we're trying to solve